src/main/js/controls/Bubble/helpers/helpers.js

```diff
@@ -299,9 +282,8 @@ const removeBubbleBlur = (nodes) =>
  */
 const blurActionHandler = (target, hoverState) => {
     d3.select(target).attr("aria-selected", true);
-    const allCircleNodes = getAllCircleNodes();
-    if (hoverState === constants.HOVER_EVENT.MOUSE_ENTER) {
-        enforceBubbleBlur(allCircleNodes);
+    if (hoverState === constants.MOUSE_EVENT.MOUSE_ENTER) {
```

Comment:
Why do you need this? Are we tracking any hovers?

Reply:
I will remove this it's not needed.